applications/rag/main.tf

```diff
@@ -175,10 +186,19 @@ module "cloudsql" {
   providers     = { kubernetes = kubernetes.rag }
   project_id    = var.project_id
   instance_name = local.cloudsql_instance
-  namespace     = local.kubernetes_namespace
   region        = local.cloudsql_instance_region
   network_name  = local.network_name
-  depends_on    = [module.namespace]
+  depends_on    = [module.network]
+}
+
+module "cloudsql-secret" {
```

Comment:
I don't think this needs to be a module, it's unlikely to be re-used. Can we just make a resource here instead?